python/lsst/obs/decam/decamMapper.py

```diff
@@ -358,12 +358,13 @@ def getLinearizerDir(cls):
     def map_linearizer(self, dataId, write=False):
         """Map a linearizer"""
         actualId = self._transformId(dataId)
-        location = os.path.join(self.getLinearizerDir(), "%02d.fits" % (dataId["ccdnum"]),)
+        location = "%02d.fits" % (dataId["ccdnum"])
```

Comment:
My guess is that the last comma that was dropped was supposed to be one level deeper:

    os.path.join(self.getLinearizerDir(), "%02d.fits" % (dataId["ccdnum"],))

If you drop that you can also drop parens around `dataId` or replace whole thing with `.format()`